orchestrator-salt/src/main/resources/salt-common/salt/monitoring/cdp-prometheus.sls

```diff
@@ -76,6 +76,13 @@ remove_prometheus_pwd:
   {%- endif %}
 {%- endif %}
 
+/var/log/cdp-prometheus/cdp-prometheus.log:
+  file.managed:
+    - user: "root"
+    - group: "root"
+    - makedirs: True
+    - mode: 644
```

Comment:
why is the necessary?

Reply:
The start command would fail, if the log file didn't exist. (see the start command in the service's unit file cdp-prometheus.service.j2)

Reply:
I don't see why would it fail. maybe the directory should be created but not the log file

Reply:
I changed it so only the directory is created ahead of time.